RELEASE_NOTES.md

```diff
@@ -1,3 +1,10 @@
+### 0.11.3-beta
```

Comment:
A .12 might not be unreasonable as there are new supported expression features (and a removal of an obsoletion)